lib/aur-chroot

```diff
@@ -77,7 +81,8 @@ makepkg_conf=${makepkg_conf-/usr/share/devtools/makepkg-$machine.conf}
 
 # FIXME: local repos may contain multiple file:// entries
 for repo in "${host_repo[@]}"; do
-    server=$(get_conf_repo "$repo" | tee "$tmp"/custom.conf | grep -Em1 '^file://' || true)
```

Comment:
Just updating the regex from `grep` to either of these would also work:
```suggestion
    server=$(get_conf_repo "$repo" | tee "$tmp"/custom.conf | grep -oEm1 'file://.*$' || true)
```
```suggestion
    server=$(get_conf_repo "$repo" | tee "$tmp"/custom.conf | grep -Pom1 '(?<=^Server = )file://.*$'
```

Reply:
Ah, that's cleaner. Didn't think of `grep -o`.